wechat/models.py

```diff
@@ -33,6 +33,13 @@ class Activity(models.Model):
     STATUS_SAVED = 0
     STATUS_PUBLISHED = 1
 
+    @classmethod
+    def get_by_id(cls, activity_id):
+        try:
+            return cls.objects.get(id=activity_id)
```

Comment:
I think use 'filiter' is better. u can use .first() to get  the first item. use 'get' may create error when return none.